docs/FAQ.md

```diff
@@ -10,16 +10,13 @@ OpenCost is a vendor-neutral open source project for measuring and allocating in
 The project combines the Kubecost core allocation engine with a new community-driven [spec](https://github.com/kubecost/opencost/tree/develop/spec) on how to monitor costs in Kubernetes land. The allocation engine is a Golang implementation that conforms to spec. Both have now been submitted to the CNCF as a [sandbox project](https://www.cncf.io/sandbox-projects/).
 
 ### How do I get started with the project?
-You can [install](https://docs.kubecost.com/install) the project in minutes. Come join the [Slack](https://join.slack.com/t/kubecost/shared_invite/enQtNTA2MjQ1NDUyODE5LWFjYzIzNWE4MDkzMmUyZGU4NjkwMzMyMjIyM2E0NGNmYjExZjBiNjk1YzY5ZDI0ZTNhZDg4NjlkMGRkYzFlZTU) community if you have more questions, have input, or want to get involved in other ways. 
+You can [install](install.md) the project in minutes. Come join the [Slack](https://join.slack.com/t/kubecost/shared_invite/enQtNTA2MjQ1NDUyODE5LWFjYzIzNWE4MDkzMmUyZGU4NjkwMzMyMjIyM2E0NGNmYjExZjBiNjk1YzY5ZDI0ZTNhZDg4NjlkMGRkYzFlZTU) community if you have more questions, have input, or want to get involved in other ways. 
 
 ### Do I need to pay to use OpenCost?
-No. OpenCost is open source and free to use.
+No. OpenCost is open source (Apache 2.0) and free to use.
 
 ### How does OpenCost relate to Kubecost?
-OpenCost is the cost allocation engine originally built by Kubecost. It’s actively used in the free Kubecost community version up to the Enterprise Kubecost version for cost allocation. However, Kubecost, along with a group of founding members, announced intentions to onboard OpenCost as a CNCF sandbox project. If the CNCF accepts our onboarding application, the CNCF will become the governing body for OpenCost moving forward.
-
-### Why do I see Kubecost mentioned within OpenCost?
```

Comment:
I think this is better mentioned on the Kubecost website. It just feels like this FAQ is referencing Kubecost a lot...

Reply:
agreed!